tripal_chado/src/api/tripal_chado.query.api.inc

```diff
@@ -1310,28 +1318,41 @@ function chado_select_record($table, $columns, $values, $options = NULL, $chado_
 
   // Check that our columns and values arguments are proper arrays.
   if (!is_array($columns)) {
-    tripal_report_error('tripal_chado', TRIPAL_ERROR,
-      'chado_select_record; the $columns argument must be an array. Columns:%columns',
-      ['%columns' => print_r($columns, TRUE)],
-      ['print' => $print_errors]
+	  // tripal_report_error('tripal_chado', TRIPAL_ERROR,
+    //   'chado_select_record; the $columns argument must be an array. Columns:%columns',
+    //   ['%columns' => print_r($columns, TRUE)],
+    //   ['print' => $print_errors]
+    // );
+    $logger->error('
+      tripal_chado error chado_select_record; the $columns argument must be an array. 
+      Columns:' . print_r($columns, TRUE) .  $print_errors
     );
     return FALSE;
   }
   if (!is_array($values)) {
-    tripal_report_error('tripal_chado', TRIPAL_ERROR,
-      'chado_select_record; the $values argument must be an array. Values:%values',
-      ['%values' => print_r($values, TRUE)],
-      ['print' => $print_errors]
+    // tripal_report_error('tripal_chado', TRIPAL_ERROR,
+    //   'chado_select_record; the $values argument must be an array. Values:%values',
+    //   ['%values' => print_r($values, TRUE)],
+    //   ['print' => $print_errors]
+    // );
+    $logger->error('
+      tripal_chado error chado_select_record; the $values argument must be an array. 
+      Values:' . print_r($values, TRUE) . $print_errors
     );
     return FALSE;
   }
 
   // Get the table description.
+  // echo 'chado_select_record Table:' . $table . "\n";
   $table_desc = chado_get_schema($table, $chado_schema_name);
+  // echo 'chado_select_record ' . $table . ' ' . print_r($table_desc, TRUE) . "\n";
```

Comment:
Please remove debugging code. This branch will be merged into 9.x-4.x since it is not importer specific and we want to minimize divergence so we need to clean it up :-)

Reply:
Removed these comments as well, thanks! Will be seen in the upcoming commit.